src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -511,8 +583,10 @@ static void cc_destroy_channel_elem(grpc_channel_element* elem) {
   chand->method_params_table.reset();
   grpc_client_channel_stop_backup_polling(chand->interested_parties);
   grpc_pollset_set_destroy(chand->interested_parties);
+  GRPC_COMBINER_UNREF(chand->data_plane_combiner, "client_channel");
   GRPC_COMBINER_UNREF(chand->combiner, "client_channel");
-  GRPC_ERROR_UNREF(chand->disconnect_error);
+  GRPC_ERROR_UNREF(
```

Comment:
Destroy can also be relaxed because we cannot call Destroy in parallel to CompareExchange and it has to be ordered. So, I would suggest using RELAXED here.